subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -127,6 +127,7 @@ choice NRF_CLOUD_CREDENTIALS_MGMT
 config NRF_CLOUD_CREDENTIALS_MGMT_MODEM
 	depends on NRF_MODEM_LIB
 	select MODEM_KEY_MGMT
+	select AT_SHELL_UNESCAPE_LF if AT_SHELL
```

Comment:
Perhaps 

```
imply AT_SHELL_UNESCAPE_LF if AT_SHELL
```

although I don't know whether `imply` actually supports `if`

If not, then

```
depends on AT_SHELL_UNESCAPE_LF if AT_SHELL
```

Reply:
I would rather not have to change a bunch of overlays to get the benefit from this.

Reply:
To me, that would be ideal -- conditional selections like this can be tricky to track down. But setting that aside, does the imply alternative at least work?

Reply:
It's needed for any method of injecting modem certs when there is a shell. I am choosing to limit the scope of the change to nRF Cloud only, though I could imagine users of AWS or Azure might want it. I will leave that decision to @simensrostad.

Reply:
Yes, `imply` works fine.